internals/secrethub/mlock.go

```diff
@@ -3,12 +3,17 @@ package secrethub
 import (
 	"strconv"
 
+	"github.com/secrethub/secrethub-cli/internals/cli"
 	"github.com/secrethub/secrethub-cli/internals/cli/mlock"
 )
 
 // mlockFlag configures locking memory.
```

Comment:
Code simplified and now it is hidden.

Reply:
This indeed makes things shorter, but it also introduces a problem with processing the environment variables because of the order in which the different `PreRuns` are executed (try `SECRETHUB_NO_COLOR=1 secrethub tree <path>`, it does not work). 

This can be solved by processing the values of global flags from environmennt variables in a `PeristentPreRunE` on the root command. I have committed this suggestion.